internal/rmx.go

```diff
@@ -0,0 +1,69 @@
+package internal
+
+type MessageTyp int
+
+const (
+	Unknown = iota
+
+	Create
```

Comment:
I think these event names may need more context as we may be creating more than one type of thing (Jam, Instrument, etc). Maybe we wait until there's a conflict though.
```suggestion
	JamCreate
```